packages/picasso/src/RichText/hooks/useRichText/useRichText.tsx

```diff
@@ -64,18 +64,21 @@ const picassoMapper = (child: ReactNode): ReactNode => {
 
   const type =
     componentMap[child.type as keyof typeof componentMap] || child.type
-  const mappedChildren = child.props.children.map(picassoMapper)
+
+  const mappedChildren = child.props.children?.map(picassoMapper) ?? null
 
   return createElement(type, { key: child.key }, mappedChildren)
 }
 
 const useRichText = (value: ASTType): ReactNode[] | ReactNode => {
   const mappedTextNodes = useMemo(() => {
-    if (!value.children) {
+    const { children } = value
```

Comment:
```suggestion
    const { children: astChildren } = value
```
what about changing to this?

Reply:
I can do that 👍